src/Test.FunctionalTests.Sarif/TestData/Multitool/ValidateCommand/Inputs/SARIF1019.RuleIdMustBePresentAndConsistent_Valid.sarif

```diff
@@ -0,0 +1,79 @@
+{
+  "$schema": "https://schemastore.azurewebsites.net/schemas/json/sarif-2.1.0-rtm.5.json",
+  "version": "2.1.0",
+  "runs": [
+    {
+      "tool": {
+        "driver": {
+          "name": "Sarif Functional Testing"
+        }
+      },
+      "logicalLocations": [
+        {
+          "name": "A"
+        },
+        {
+          "name": "B"
+        }
+      ],
+      "results": [
+        {
+          "ruleId": "TST0001",
+          "message": {
+            "text": "Message."
+          },
+          "locations": [
+            {
+              "logicalLocations": [
+                {
+                  "index": 1
+                }
+              ]
+            }
+          ],
+          "rule": {
+            "index": 1
```

Comment:
>"index": 1 [](start = 12, length = 10)

The presence of the `index` property makes this an invalid SARIF file. The index is supposed to point into the array `run.tool.driver.rules`, but that array does not exist in this sample. IMO the simplest fix is to remove the `index` property from each of the three `result.rule` objects. This means that the first result will have a `result.rule` object with no properties, so just remove it.

I'm surprised this test passes, because I thought we had a validation rule that requires all "index" properties to be consistent with the arrays they refer to.